cilium-parameters.schema.cue

```diff
@@ -0,0 +1,10 @@
+package holos
+
+// the cilium addon should only refer to these parameters.  This schema
+// definition defines the interface between the platform and the addon.
+#CiliumParameters: {
+	fleet: name:   string
+	fleet: meshed: bool
+	cluster: name: string
+	cluster: os:   string
+}
```

Comment:
Definition of the component addon's interface is here, the `#CiliumParameters` definition.